sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_constants.py

```diff
@@ -17,6 +17,7 @@
 KubernetesEnvironmentVariable = "KUBERNETES_PORT"
 ServiceFabricEnvironmentVariable = "Fabric_NodeName"  # cspell:disable-line
 
+DISABLE_APPCONFIGURATION_DISCOVERY = "AZURE_APP_CONFIGURATION_DISABLE_DISCOVERY"
```

Comment:
Why is this an environment variable instead of an option/kwarg?

Reply:
This came from a discussion with the SDK team.

Reply:
Would `AZURE_APP_CONFIGURATION_DISABLE_REPLICA_DISCOVERY` be better?

Reply:
Are we going to make this an option instead of env var as discussed in the email thread?